drivers/mpathcount.py

```diff
@@ -24,6 +24,19 @@
 import glob
 import json
 
+supported = ['iscsi','lvmoiscsi','rawhba','lvmohba', 'ocfsohba', 'ocfsoiscsi', 'netapp','lvmofcoe', 'gfs2']
```

Comment:
missing space after netapp

Reply:
Was in the original. Will fix in the commit which moves it.